app/src/main/java/com/example/wangyf/android_dokumi_test/MainActivity.java

```diff
@@ -19,6 +19,18 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+//        TextView textView = (TextView)findViewById(R.id.test1);
+//        textView.setText("aaaaaaaaaaaaaaaaaaaa");
+//
+//
+//        if (true){
+//            Integer i = null;
+//            i.toString();
+//        }
+
+        SharedPreferences.Editor editor = PreferenceManager.getDefaultSharedPreferences(this).edit();
+        editor.commit();
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wang6793453/android-dokumi-test/blob/c97f6d893edc52596a1a418e59049408a1d2384a/app/src/main/java/com/example/wangyf/android_dokumi_test/MainActivity.java#L32"></span>[ApplySharedPref] Consider using <code>apply()</code> instead; <code>commit</code> writes its data to persistent storage immediately, whereas <code>apply</code> will handle it in the background<br />
 Consider using <code>apply()</code> instead of <code>commit</code> on shared preferences. Whereas <code>commit</code> blocks and writes its data to persistent storage immediately, <code>apply</code> will handle it in the background.</td>
    </tr>
  </tbody>
</table>